plugin-guide.md

```diff
@@ -134,13 +161,13 @@ and `ca.on.oicr.gsi.vidarr.WorkflowEngine`. These plugins are responsible for
 running workflows and collecting the output from the workflow. A workflow
 engine can support multiple languages (see
 `ca.on.oicr.gsi.vidarr.WorkflowLanguage` for a complete list) and choose which
-subset it can accommodate.
+subset it can support.
```

Comment:
I'm sorry, this is actually more confusing :pensive: 'A workflow engine can support multiple languages and choose which subset it can support'? that sounds like you're filtering the list twice? Would simply 'A workflow engine can support multiple languages' make more sense?